docs/source/documentation/index.html.md.erb

```diff
@@ -8,6 +8,10 @@ weight: 0
 
 # <%= current_page.data.title %>
 
+## Documentation
+
+- [Data Platform documentation](/documentation/platform/index.html)
+
 ## Architecture
 
 - [Architecture Decision Records (ADR)](/documentation/adrs/adr-index.html)
```

Comment:
ADRs now live here: https://technical-documentation.data-platform.service.justice.gov.uk/documentation/adrs/adr-index.html

Reply:
The ADRs link isn't new, it was already there and it works on the published site. I followed the same approach for the platform documentation hoping that it works.